Release Notes & new link Anti-fraud debit cards (#565)
vtexdocs/dev-portal-content: docs/guides/Payments/payments-configuration-guides.md

---

> The articles in this subsection will present the necessary steps to make payment settings.

Reviewer: ```suggestion
The articles listed below will present the necessary steps to make payment settings.
```

> | Article | Description                                                                                                                  |
> | :-----------------------------------------------: | ------------------------------------------------------------------------------------------------------------------------ |
> |                 [Setting up the type of interest rate](https://developers.vtex.com/docs/guides/setting-up-the-type-of-interest-rate)                 | How to identify the payment methods' ID and setting up the interest rate type using API.                                                                      |
> |                 [Setting up Merchant ID  in Apple Pay](https://developers.vtex.com/docs/guides/setting-up-merchant-id-in-apple-pay)                 | How to setting Merchand ID in Apple Pay.                                                                                                      |
> |                 [Cardholder document configuration](https://developers.vtex.com/docs/guides/cardholder-document-configuration)                 | How to configure cardholder document field display at checkout.                                                                 |
> |                 [Implementing a pre-analysis anti-fraud flow for debit card transactions](https://developers.vtex.com/docs/guides/implementing-a-pre-analysis-antifraud-flow-for-debit-card-transactions)                 | How to activate the anti-fraud flow for debit card transactions.                                                                           |

Reviewer: ```suggestion
| Article | Description |
| - | - |
| [Setting up the type of interest rate](https://developers.vtex.com/docs/guides/setting-up-the-type-of-interest-rate) | How to identify the payment methods' ID and set up the interest rate type using the Payments Gateway API. |
| [Setting up Merchant ID in Apple Pay](https://developers.vtex.com/docs/guides/setting-up-merchant-id-in-apple-pay) | How to set up Merchant ID in Apple Pay. |
| [Cardholder document configuration](https://developers.vtex.com/docs/guides/cardholder-document-configuration) | How to configure a cardholder document field to be displayed at checkout. |
| [Implementing a pre-analysis anti-fraud flow for debit card transactions](https://developers.vtex.com/docs/guides/implementing-a-pre-analysis-antifraud-flow-for-debit-card-transactions) | How to activate the anti-fraud flow for debit card transactions. |
```
